src/filehandler.ts

```diff
@@ -4,38 +4,48 @@ import {
 import path from 'node:path';
 import { ItemExistanceError } from './fixtureindex';
 
-export const storageDirectory = path.resolve(__dirname, '../.fixturelibrary');
-
-export async function readJsonFile(filename: string): Promise<any> {
-  // Preparing path
-  let filePath = `${storageDirectory}/${filename}`;
-  if (!filename.endsWith('.json')) filePath += '.json';
-  // Checking if file exists
-  if (!await pathExists(filePath)) {
-    throw new ItemExistanceError(`${filePath} doesn't exist!`);
+class FileHandler {
+  // eslint-disable-next-line class-methods-use-this
+  public get storageDirectory() {
+    return process.env.OFL_INDEX ?? path.resolve(__dirname, '../.fixturelibrary');
```

Comment:
worth adding a line in the readme for future people to not have to dive into the code to find this?